DESCRIPTION

```diff
@@ -30,9 +30,6 @@ Suggests:
     httptest,
     magrittr,
     spelling
-Remotes:
-    Crunch-io/rcrunch
 RoxygenNote: 6.0.1
-Encoding: UTF-8
```

Comment:
Did this give an error on check? I remember there _was_ a bug in devtools/testthat somewhere that would pop up if we didn't have this.